scale/job/models.py

```diff
@@ -2460,13 +2465,16 @@ def create_job_type_v6(self, manifest_dict, trigger_rule_dict=None, configuratio
                 raise Exception('%s is not an editable field' % field_name)
         self._validate_job_type_fields(**kwargs)
 
-        trigger_rule = self._create_seed_job_trigger_rule(manifest, trigger_rule_dict)
+        trigger_rule = None
+        if trigger_rule_dict:
+            trigger_rule = self._create_seed_job_trigger_rule(manifest, trigger_rule_dict)
 
         secrets = None
         if configuration_dict:
             configuration = JobConfigurationV6(configuration_dict, do_validate=True).get_configuration()
```

Comment:
I recommend moving this line, `configuration = JobConfigurationV6(configuration_dict, do_validate=True).get_configuration()` into the view (within a try/except for `InvalidConfiguration`) and pass the general configuration object into this method instead of passing the dict.